alpakka-sample-file-to-elasticsearch/src/main/scala/samples/javadsl/RunOps.scala

```diff
@@ -2,11 +2,11 @@ package samples.javadsl
 import java.nio.file.Path
 import java.util.concurrent.CompletionStage
 
-import akka.Done
-import akka.actor.typed.ActorSystem
+import org.apache.pekko.Done
+import org.apache.pekko.actor.typed.ActorSystem
 import samples.scaladsl
 
-import scala.compat.java8.FutureConverters._
+import scala.jdk.FutureConverters._
```

Comment:
please don't use this class - use the Pekko one

Reply:
Actually if we are talking about samples, they should be using either `scala.jdk.FutureConverters._` or `scala.compat.java8.FutureConverters._` depending on what Scala version the samples are compiled for.

The pekko import's are only for internal Pekko code which I assume is not what the samples are for.